src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -2812,11 +2856,21 @@ namespace winrt::Microsoft::Terminal::Control::implementation
                     allPathsString += fullPath;
                 }
 
-                _core.PasteText(winrt::hstring{ allPathsString });
+                _pasteTextWithBroadcast(winrt::hstring{ allPathsString });
             }
         }
     }
 
+    void TermControl::_pasteTextWithBroadcast(const winrt::hstring& text)

```

Comment:
This name really confused me - it actually ISN'T used in paste(!) but it made me double-take because we are manually broadcasting the paste event and the incoming text doesn't come back through here. Does it? Or, does it...

Reply:
(Since I was re-requested for review) _Does_ this have anything to do with pasting? It looks like it doesn't.